get.sh

```diff
@@ -626,40 +628,55 @@ fi
 checkRepoSHA()
 {
 	output_file="$TESTDIR/TKG/SHA.txt"
-	echo "$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file"
-	$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file
+	if [ -e ${output_file} ]; then
+		echo "rm $output_file"
+		rm ${output_file}
+	fi
```

Comment:
We cannot remove `SHA.txt` every time we check the SHA. Also, we are generating `testenv.properties` file, we should not remove `SHA.txt`.